Clarify some things in documentation (#49)
Lonami/pyndustric: README.md

Screen.flush()
```

Built-in properties to access the environment, like time or links:
Built-in properties to access the environment, like counter or time:

```python
counter = Env.counter
current_time = Env.time

> link_count = Env.link_count

Author: I believe it can create confusion from people that are coming from mlog. I certainly saw that, thought about how it is `@links` in mindustry and my brain automaticaly striked out possibility to write anything that starts with `@` using `Env`. It took me to dive into source to figure out that compiler would happily write anything after `Env.` as variable that starts with `@` instad of just what is defined stuff in pyndustri.pyi and enforce it.

compiles to:

```
set counter @counter
set current_time @time
```

> Beyond defined ones in [`pyndustri.pyi`], this is also the way to write mindustry built-in variables and constants (ones that start with @). If written as string literals, compiled result will be a string, not a variable. E.g:

Reviewer: For very long lines I try to stay below 100 characters per line:

```suggestion
Beyond the ones defined in [`pyndustri.pyi`], this is also the way to write Mindustry built-in variables and constants
(those that start with @). If written as string literals, compiled result will be a string, not a variable. For example:
```

```python
titanium = "@titanium" # set titanium "@titanium"
titanium = @titanium # compile error

> titanium = Env.titanium # set titanium @titanium

Reviewer: Actually these "built-in" variables are defined for the most part in `Senseable`. Perhaps the `Env` could subclass `Senseable` so that it these type checked?

Author: `Senseable` works only as property of something: `a = container1.titanium` -> `sensor a container1 @titanium` 
I needed a way to write builtin variables outside of sensor: `Unit.build(0, 0, Env.sorter, 0, Env.titanium) `
I also think the end user should be ablo to write arbitrarily Mindustry built-ins in some way. First I thought about something like: `built-in("@sorter")` (I tried other compilers) until I figured out that Env goes further than what is written in `pyndustri.pyi` (had to read source code to figure that out, I don't know why it wasn't obvious), had "Ahaaa" moment and decided to write it in docs

Alternativly, you can run it as module without installation by placing your program file at the root of the project directory.

To compile your program, run the [`pyndustric`] module and pass the files to compile as input arguments.
`-` can be used as a file to refer to standard input:

```sh
$ python -m pyndustric yourprogram.py
```

> The compiled program will be printed to standard output which can be either copeid or redirected to a file:
> 
> ```
> $ python -m pyndustric yourprogram.py > yourprogram.mlog
> ```

Reviewer: The redirection to files should be obvious to people who have some experience with the shell, so I tried to explain it a bit more for those who actually need this information ("it creates a new file").

Also might as well document the `-c` option. Perhaps the `setup.py` should be updated to support an optional `pip install pyndustric[autocopy]` or something (not in this PR though).

````suggestion
The compiled program will be printed to standard output by default.
You can also redirect the output to a file to create a new file with the compiled program:

```sh
$ python -m pyndustric yourprogram.py > yourprogram.mlog
```

If the optional dependency `autoit` is installed, `-c` or `--clipboard` can be used to
automatically copy the code to the clipboard which allows for very fast edit cycles:

```sh
$ python -m pyndustric -c yourprogram.py
```
````
